vectorsearch/indices/faiss-index.json

```diff
@@ -21,6 +21,9 @@
         "target_field": {
           "type": "knn_vector",
           "dimension": {{ target_index_dimension }},
+          {%- if mode is defined %}
+          "mode": "{{ mode }}"
```

Comment:
i think you might need comma at end. Did you test this workload manually? can you post result summary with mode?

Reply:
Thanks for Catching that , It works as expected in both Cases , With and Without Mode , Updated With Both Test Cases in Pr Description.